src/amchart/Area.js

```diff
@@ -6,35 +11,73 @@
         root.amchart_Area = factory(root.d3, root.amchart_CommonSerial, root.amcharts, root.api_INDChart);
     }
 }(this, function(d3, CommonSerial, AmCharts, INDChart) {
+    /**
+     * @class amchart_Area
+     * @extends amchart_CommonSerial
+     * @extends api_INDChart
+     * @implements api_INDChart
+     */
     function Area() {
         CommonSerial.call(this);
+        /**
+         * Specifies the class name of the container.
+         * @member {string} _class
+         * @memberof amchart_Area
+         * @private
+         */
         this._class = "amchart_Area";
+        /**
+         * Specifies the HTML tag type of the container.
+         * @member {string} _tag
+         * @memberof amchart_Area
+         * @private
+         */
         this._tag = "div";
-
+        /**
+         * Specifies the graph type of the AmChart Widget.
+         * @member {string} _gType
+         * @memberof amchart_Area
+         * @private
+         */
         this._gType = "line";
     }
 
     Area.prototype = Object.create(CommonSerial.prototype);
     Area.prototype.implements(INDChart.prototype);
 
-    /**
-     * Publish Params Common To Other Libraries
-     */
+    // Publish Params Common To Other Libraries
+
     Area.prototype.publish("paletteID", "default", "set", "Palette ID", Area.prototype._palette.switch(),{tags:['Basic','Shared']});
     Area.prototype.publish("isStacked", false, "boolean", "Stack Chart",null,{tags:['Basic','Shared']});
     Area.prototype.publish("fillOpacity", 0.7, "number", "Opacity of The Fill Color", null, {min:0,max:1,step:0.001,inputType:'range',tags:['Intermediate','Shared']});
 
+    // Publish Params Unique To This Widget
 
-    /**
-     * Publish Params Unique To This Widget
-     */
     Area.prototype.publish("tooltipTemplate","[[category]]: [[value]]", "string", "Tooltip Text",null,{tags:['Intermediate']});
     Area.prototype.publish("stackType", "regular", "set", "Stack Type",["none","regular","100%"],{tags:['Basic']});
 
+    /**
+     * The function that is executed on first render.
+     * @method enter
+     * @memberof amchart_Area
+     * @instance
```

Comment:
if you want your methods to show up in your class and propagate to its children you have to tell it is an instance method/variable etc...(same goes for your interfaces)